cfgov/unprocessed/js/apps/financial-well-being/fwb-questions.js

```diff
@@ -1,5 +1,3 @@
-'use strict';
```

Comment:
Funny that 4 removes this, I feel like I was constantly adding it in mindlessly.

Reply:
It's not something that's gone now, but is not needed inside modules. [This setting](https://github.com/cfpb/cfgov-refresh/pull/3660/files#diff-1dc6ee56b778cd91e0327b52aaeaa1b9R5) can either be `module` or `script`, as module it won't include use strict inside modules.